week-1/exercises/paths.js

```diff
@@ -16,3 +16,7 @@ console.log(`The base part of ${filePath} is ${base}`);
 
 // Create a variable to store the dir part of the filePath variable
 // Create a variable to store the ext part of the variable
```

Comment:
Looks like you might have missed this requirement to get the ext part of the variable? 🤔

Reply:
I have added this requirement